diff --git a/node_modules/@openzeppelin/guardian-client/dist/conversion.js b/node_modules/@openzeppelin/guardian-client/dist/conversion.js
index f0a1866..3caac77 100644
--- a/node_modules/@openzeppelin/guardian-client/dist/conversion.js
+++ b/node_modules/@openzeppelin/guardian-client/dist/conversion.js
@@ -67,7 +67,8 @@ export function fromServerProposalMetadata(server) {
 export function fromServerDeltaObject(server) {
     const { txSummary, signatures, metadata } = extractDeltaPayload(server.delta_payload);
     return {
-        accountId: server.account_id,
+        metadata: server.metadata ? { proposal: server.metadata.proposal ? fromServerProposalMetadata(server.metadata.proposal) : undefined } : undefined,
+    accountId: server.account_id,
         nonce: server.nonce,
         prevCommitment: server.prev_commitment,
         newCommitment: server.new_commitment,
diff --git a/node_modules/@openzeppelin/guardian-client/dist/server-types.d.ts b/node_modules/@openzeppelin/guardian-client/dist/server-types.d.ts
index 2c741e8..5c680ea 100644
--- a/node_modules/@openzeppelin/guardian-client/dist/server-types.d.ts
+++ b/node_modules/@openzeppelin/guardian-client/dist/server-types.d.ts
@@ -63,6 +63,7 @@ export interface ServerProposalMetadata {
     timelock_height?: number;
 }
 export interface ServerDeltaObject {
+    metadata?: { proposal?: ServerProposalMetadata };
     account_id: string;
     nonce: number;
     prev_commitment: string;
diff --git a/node_modules/@openzeppelin/guardian-client/dist/types.d.ts b/node_modules/@openzeppelin/guardian-client/dist/types.d.ts
index e98eb39..e8ca9f6 100644
--- a/node_modules/@openzeppelin/guardian-client/dist/types.d.ts
+++ b/node_modules/@openzeppelin/guardian-client/dist/types.d.ts
@@ -108,6 +108,7 @@ export interface ProposalMetadata {
     timelockHeight?: number;
 }
 export interface DeltaObject {
+    metadata?: { proposal?: ProposalMetadata };
     accountId: string;
     nonce: number;
     prevCommitment: string;
diff --git a/node_modules/@openzeppelin/guardian-client/src/conversion.ts b/node_modules/@openzeppelin/guardian-client/src/conversion.ts
index 1c734a7..33c5863 100644
--- a/node_modules/@openzeppelin/guardian-client/src/conversion.ts
+++ b/node_modules/@openzeppelin/guardian-client/src/conversion.ts
@@ -128,6 +128,7 @@ export function fromServerDeltaObject(server: ServerDeltaObject): DeltaObject {
   const { txSummary, signatures, metadata } = extractDeltaPayload(server.delta_payload as ServerDeltaPayload);
 
   return {
+    metadata: server.metadata ? { proposal: server.metadata.proposal ? fromServerProposalMetadata(server.metadata.proposal) : undefined } : undefined,
     accountId: server.account_id,
     nonce: server.nonce,
     prevCommitment: server.prev_commitment,
diff --git a/node_modules/@openzeppelin/guardian-client/src/server-types.ts b/node_modules/@openzeppelin/guardian-client/src/server-types.ts
index a53ffb7..24ed3d4 100644
--- a/node_modules/@openzeppelin/guardian-client/src/server-types.ts
+++ b/node_modules/@openzeppelin/guardian-client/src/server-types.ts
@@ -67,6 +67,7 @@ export interface ServerProposalMetadata {
 }
 
 export interface ServerDeltaObject {
+    metadata?: { proposal?: ServerProposalMetadata };
   account_id: string;
   nonce: number;
   prev_commitment: string;
diff --git a/node_modules/@openzeppelin/guardian-client/src/types.ts b/node_modules/@openzeppelin/guardian-client/src/types.ts
index bcf75bc..9af5b47 100644
--- a/node_modules/@openzeppelin/guardian-client/src/types.ts
+++ b/node_modules/@openzeppelin/guardian-client/src/types.ts
@@ -125,6 +125,7 @@ export interface ProposalMetadata {
 }
 
 export interface DeltaObject {
+    metadata?: { proposal?: ProposalMetadata };
   accountId: string;
   nonce: number;
   prevCommitment: string;
